libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/NavigationSession.kt

```diff
@@ -120,3 +139,42 @@ sealed class NavigationSessionState {
         override val sessionId: String
     ) : NavigationSessionState()
 }
+
+sealed class NavigationSessionStateV2 {
```

Comment:
how to you think, should we replace sealed class by a set of string constants, to be able to add more states in the future?

Reply:
I tend to think no. I doubt that we will need more states before v3. And for users it should be more comfortable to use sealed class.